src/hmpps/types.ts

```diff
@@ -0,0 +1,9 @@
+export type designSystemEnv = 'local' | 'dev' | 'pre' | 'prod'
+export class ServiceHeaderConfig {
```

Comment:
could we see what this would look like it it were an interface rather than a class? I wouldn't expect any functions to exist in the config (though welcome to be proven wrong) and try to follow https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/server/%40types/express/index.d.ts as best as we could